libs/ui/src/display_settings/tab_bar.tsx

```diff
@@ -23,7 +24,7 @@ const Container = styled.div<ContainerProps>`
   gap: max(0.25rem, ${(p) => p.theme.sizes.minTouchAreaSeparationPx}px);
 
   & > * {
-    max-width: ${100 / PANE_IDS.length}%;
+    max-width: ${(p) => 100 / p.tabCount}%;
```

Comment:
unrelated fix for tabs getting unnecessarily shortened when there are only 2 tabs displayed